peercrawler_html_server.py

```diff
@@ -125,20 +125,22 @@ def parse_args():
     parser.add_argument("-d", "--delay", type=int, default=300,
                         help="delay between crawls in seconds")
     parser.add_argument("-l", "--nolisten", action="store_true", default=False,
-                        help="listen to incoming connections")
+                        help="disable incoming connection listener")
     parser.add_argument("-p", "--port", type=int, default=5001,
                         help="port to listen on for incoming requests")
```

Comment:
This could say that http just to make it obvius it is not setting the peercrawler port.
It woudl also be nice to be able to specify the peercrawler from the command line

Reply:
I'm not sure what you mean by "specify the peercrawler from the command line".

Reply:
I think I meant write, 
It would also be nice to be able to specify the peercrawler listening port from the command line.